pkg/apis/config/v1alpha4/types.go

```diff
@@ -85,6 +85,28 @@ type Node struct {
 	// ExtraPortMappings describes additional port mappings for the node container
 	// binded to a host Port
 	ExtraPortMappings []PortMapping `yaml:"extraPortMappings,omitempty"`
+
+	// KubeadmConfigPatches are applied to the generated kubeadm config as
```

Comment:
These should document their relationship to the cluster wide patches, IE that they are applied next and are more specific.

Reply:
Agreed. At first I thought about struct embedding, but I honestly don't know if it'll make the code and docs more confusing.

Reply:
we just need a comment here about the behavior here (being applied on top of the others)

Reply:
Cool beans! I'll work on these changes later today. I'll be integration testing this quite a bit as I'm working on an issue for [kube-bench](https://github.com/aquasecurity/kube-bench/issues/439) 😄

Reply:
thanks :-)